### DISCUSSION TITLE
Streaming from a start time in the past, how?

### DISCUSSION THREAD

**Discussion by bob-swede** [streamlink/streamlink#3921] [Q&A] [answered]
I need streamlink in order to stream from a live stream but starting at a specific time in the past.
Use case:
Recording a youtube live stream from the top of the last hour, say 39 minutes ago and lasting for 60 minutes. So it will stream from the start time until 21 minutes from now.
Can this be done and if so what would be the streamlink command to get the video stream into an mp4 file at 480p resolution?
Notice that the stream I use is 12 hours long and constantly streaming, I want to record full hours of that starting sometime in the past.

**Comment by mkbloke** [comment, marked as answer]
YouTube rewind is not supported at the moment.  #2936

**Reply by bob-swede** [reply]
Too bad!
Then I have to set up a system to constantly download the stream in some decently sized chunks and postprocess these if needed. Requires an unmetered Internet connection, though...
